app/controllers/lessons_controller.rb

```diff
@@ -0,0 +1,29 @@
+class LessonsController < ApplicationController
+  before_action :single_lesson, only: [:create]
+
+  def create
+    @lesson = Lesson.new(lesson_params)
+    if @lesson.save
+      redirect_to new_lesson_lesson_word_url(@lesson)
+    end
+  end
+
+  def show
+    @lesson = Lesson.find(params[:id])
+    @category = Category.find(@lesson.category_id)
```

Comment:
Dòng code này có thể thay thế bằng cách sử dụng association nha em

Reply:
da thay the sao a